From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igor.pecovnik@gmail.com>
Date: Fri, 8 Jan 2016 17:03:44 +0100
Subject: [ARCHEOLOGY] Patches for Marvell Armada, kernel 4.3 and 4.4

> X-Git-Archeology: > recovered message: > Credits: Russell King http://www.home.arm.linux.org.uk/~rmk/clearfog/
> X-Git-Archeology: - Revision 06fe0ffe8e197c58ca9d2d919b050311e6a05411: https://github.com/armbian/build/commit/06fe0ffe8e197c58ca9d2d919b050311e6a05411
> X-Git-Archeology:   Date: Fri, 08 Jan 2016 17:03:44 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Patches for Marvell Armada, kernel 4.3 and 4.4
> X-Git-Archeology:
> X-Git-Archeology: - Revision 68a6849497488aa8b9f1486c4f13ce4353c8cd53: https://github.com/armbian/build/commit/68a6849497488aa8b9f1486c4f13ce4353c8cd53
> X-Git-Archeology:   Date: Wed, 03 Feb 2016 20:54:28 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Marvel patches move from one dir to another, small fixes
> X-Git-Archeology:
> X-Git-Archeology: - Revision baab6587a50fbc75c7b593110db50796166d9648: https://github.com/armbian/build/commit/baab6587a50fbc75c7b593110db50796166d9648
> X-Git-Archeology:   Date: Fri, 09 Dec 2016 21:09:22 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Refactor Marvell kernel sources
> X-Git-Archeology:
> X-Git-Archeology: - Revision e31425190fa09431c96067ab7371d72a9852e282: https://github.com/armbian/build/commit/e31425190fa09431c96067ab7371d72a9852e282
> X-Git-Archeology:   Date: Fri, 17 Aug 2018 09:17:19 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Move MVEBU, Clearfog & Helios4, DEVelopment branch to 4.18.y ... removed only obviously unneded patches, while the rest needs some/a lot of rework to meet current NEXT levels. http://ix.io/1kpE Tested also without our patchset - severe problems on network stack.
> X-Git-Archeology:
> X-Git-Archeology: - Revision e261c6f82835bd9b12e07ba837b55fbf1aaa4327: https://github.com/armbian/build/commit/e261c6f82835bd9b12e07ba837b55fbf1aaa4327
> X-Git-Archeology:   Date: Wed, 31 Jul 2019 12:51:00 +0200
> X-Git-Archeology:   From: Aditya Prayoga <aprayoga@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move mvebu DEFAULT, NEXT and DEV branch to next kernel (LTS) and U-boot #1426 (#1487)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision aa3d60f57e84d02887c63cae176bdec96b560e38: https://github.com/armbian/build/commit/aa3d60f57e84d02887c63cae176bdec96b560e38
> X-Git-Archeology:   Date: Thu, 10 Dec 2020 11:47:33 +0100
> X-Git-Archeology:   From: Rosen Penev <rosenp@gmail.com>
> X-Git-Archeology:   Subject: refreshed mvebu with quilt (#2419)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 5b1c1c2897a570c173c40204e98257b2dd7a74c9: https://github.com/armbian/build/commit/5b1c1c2897a570c173c40204e98257b2dd7a74c9
> X-Git-Archeology:   Date: Thu, 04 Jan 2024 00:06:37 +0530
> X-Git-Archeology:   From: Lane Jennison <lane@lane-fu.com>
> X-Git-Archeology:   Subject: mvebu-edge: move to 6.6.y
> X-Git-Archeology:
---
 drivers/cpuidle/cpuidle-mvebu-v7.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/drivers/cpuidle/cpuidle-mvebu-v7.c b/drivers/cpuidle/cpuidle-mvebu-v7.c
index 111111111111..222222222222 100644
--- a/drivers/cpuidle/cpuidle-mvebu-v7.c
+++ b/drivers/cpuidle/cpuidle-mvebu-v7.c
@@ -42,8 +42,12 @@ static __cpuidle int mvebu_v7_enter_idle(struct cpuidle_device *dev,
 
 	cpu_pm_exit();
 
+	/*
+	 * If we failed to enter the desired state, indicate that we
+	 * slept lightly.
+	 */
 	if (ret)
-		return ret;
+		return 0;
 
 	return index;
 }
-- 
Armbian

